docs/coding-style.md

```diff
@@ -40,6 +50,10 @@ Local functions should always be placed at the end of a method.
     }
 ```
 
+### Separation
```

Comment:
I moved this separately because everything above is about order, while this is about spacing